api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -91,7 +90,9 @@ message HttpConnectionManager {
       EGRESS = 1;
     }
 
-    // The span name will be derived from this field.
+    // The span name will be derived from this field. If
+    // :ref:`traffic_direction <envoy_api_msg_listener.traffic_direction>` is
+    // specified on the parent listener, then it is used instead of this field.
```

Comment:
Should this be marked as deprecated?

Reply:
I think we can.

Reply:
I think we should deprecate this rather than having this override thing.

Reply:
We can't reliably tell if the field is present or not (due to a bad choice of the default).
That means we have to respect it until we can remove it from the API (which will happen according to the regular deprecation policy).

Reply:
But you don't want to add `[deprecated = true]` to the field's options yet?

Reply:
I'm OK adding `deprecated = true`, just waiting for the input from API shepherds.

Reply:
Yep, let's deprecate, thanks.

Reply:
Done.